resource/js/form.js

```diff
@@ -1,24 +1,15 @@
-// compute duration
 function manage_duration($scope) {
     if ($scope.traffic != undefined) {
-        display_experiment_duration($scope.hide_duration);
+        display_experiment_duration($scope);
     } else {
         hide_field("duration");
     };
 }
 
 
-function display_experiment_duration() {
-    let duration = compute_duration();
+function display_experiment_duration($scope) {
```

Comment:
I would avoid passing all the scope to the function as it contains a lot of useless informations
```suggestion
function display_experiment_duration(sample_size_A, sample_size_B, traffic) {
```

Reply:
done in https://github.com/dataiku/dss-plugin-ab-testing/pull/6/commits/fe62737a557effd806f29227fc5e561f63bcb3be